yarl/_url.py

```diff
@@ -666,6 +666,31 @@ def name(self):
         """The last part of parts."""
         return self._UNQUOTER(self.raw_name)
 
+    @cached_property
+    def raw_suffix(self):
+        name = self.raw_name
+        i = name.rfind(".")
+        if 0 < i < len(name) - 1:
+            return name[i:]
+        else:
+            return ""
+
+    @cached_property
+    def suffix(self):
+        return self._UNQUOTER(self.raw_suffix)
+
+    @cached_property
+    def raw_suffixes(self):
+        name = self.raw_name
+        if name.endswith("."):
+            return []
+        name = name.lstrip(".")
+        return ["." + suffix for suffix in name.split(".")[1:]]
```

Comment:
Minor neat.
Return a tuple please here and in `suffixes`, update types in `__init__.pyi`.
`url.parts` return a tuple already, it follows the total *immutability* rule for the library.